templates/taxi/manufacturer_list.html

```diff
@@ -1,12 +1,18 @@
 {% extends "base.html" %}
+{% load crispy_forms_filters %}
 
 {% block content %}
   <h1>
     Manufacturer List
-    <a href="{% url 'taxi:manufacturer-create' %}" class="btn btn-primary link-to-page">
+    <a href="{% url 'taxi:manufacturer-create' %}"
+       class="btn btn-primary link-to-page">
       Create
     </a>
   </h1>
+  <form method="get" action="" class="form-inline">
+    {{ search_form| crispy }}
```

Comment:
There is an unnecessary space between the pipe `|` and `crispy` in the template filter `{{ search_form| crispy }}`. Remove the space to ensure the filter is applied correctly: `{{ search_form|crispy }}`.